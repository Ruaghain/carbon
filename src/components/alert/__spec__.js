import React from 'react';
import { shallow } from 'enzyme';
import Alert from './alert';

describe('Alert', () => {
  const renderShallow = () => (
    shallow(
      <Alert
        open
        onCancel={ () => {} }
        title='Alert title'
        subtitle='Alert Subtitle'
        data-element='bar'
        data-role='baz'
      />
    )
  );

  it('include correct component, element and role data tags', () => {
    expect(renderShallow()).toMatchSnapshot();
  });
});
