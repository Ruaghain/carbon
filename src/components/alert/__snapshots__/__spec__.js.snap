// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert include correct component, element and role data tags 1`] = `
<Dialog
  ariaRole="dialog"
  autoFocus={true}
  className="carbon-alert"
  componentTag="alert"
  data-element="bar"
  data-role="baz"
  onCancel={[Function]}
  open={true}
  showCloseIcon={true}
  size="extra-small"
  subtitle="Alert Subtitle"
  title="Alert title"
/>
`;
